airflow/utils/serve_logs.py

```diff
@@ -138,10 +142,14 @@ def serve_logs():
     wsgi_app = create_app()
 
     worker_log_server_port = conf.getint('logging', 'WORKER_LOG_SERVER_PORT')
-    options = {
-        'bind': f"0.0.0.0:{worker_log_server_port}",
-        'workers': 2,
-    }
+
+    # Make sure to have both an IPv4 and an IPv6 interface.
+    # Gunicorn can bind multiple addresses, see https://docs.gunicorn.org/en/stable/settings.html#bind.
+    options = [
+        GunicornOption("bind", f"0.0.0.0:{worker_log_server_port}"),
+        GunicornOption("bind", f"[::]:{worker_log_server_port}"),
```

Comment:
Does this cause any problems if ipv6 isn't enabled?

Reply:
Yes, it breaks the gunicorn startup with a failure. Below more details https://github.com/apache/airflow/pull/24846#issuecomment-1231578766